content/en/docs/releases/next-release-preview/index.md

```diff
@@ -45,3 +45,18 @@ resource:
 ### Front50
 
 https://github.com/spinnaker/front50/pull/1275 adds the `sql.healthIntervalMillis` property that controls the interval to refresh information that the /health endpoint provides.  It defaults to 30 seconds, the value before it was configurable.
+
+### Kork
+
+https://github.com/spinnaker/kork/pull/1069 adds support for artifact storage with AWS S3. This compresses `embedded/base64` types to `remote/base64` reducing the artifact size in the context. The size of the context had a reduction of 70% for larger pipelines, and 30% for smaller pipelines.
```

Comment:
Mention the other PRs too?  Or maybe [the readme](https://github.com/spinnaker/kork/blob/18d1c6e88597a9147851b37412ea38b3fd7032d5/kork-artifacts/src/main/java/com/netflix/spinnaker/kork/artifacts/README.md)?

Reply:
updated

Reply:
Or did we want release notes for each service? Cause it'd be

```
### Clouddriver
implements artifact store
```
etc.
So didn't feel it warranted its own release notes

Reply:
Yeah, I don't think we need each microservice to call it out.  That said, maybe a kork section doesn't really make sense either.  Suggested tweak coming.